src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -841,6 +843,234 @@ def test_WINNF_FT_S_SIQ_11(self):
       self.assertFalse('availableChannel' in response[response_num])
       self.assertEqual(response[response_num]['response']['responseCode'], 300)
 
+  def generate_SIQ_12_default_config(self, filename):
+    """Generates the WinnForum configuration for SIQ.12."""
+    # Load GWBL information
+    gwpz_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'gwpz_record_0.json')))
+    # Load device info
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+
+    # Load PAL records
+    pal_record_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+    pal_record_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_1.json')))
+
+    # Load PPA records
+    ppa_record1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+    ppa_record2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_1.json')))
+    
+    pal_low_frequency1 = 3570000000
+    pal_high_frequency1 = 3580000000
+    pal_user_id1 = "pal_user_1"
+    pal_low_frequency2 = 3600000000
+    pal_high_frequency2 = 3610000000
+    pal_user_id2 = "pal_user_2"
+
+    # Make PAL and PPA record consistent
+    ppa_record1, pal_records1 = makePpaAndPalRecordsConsistent(
+        ppa_record1, [pal_record_0], pal_low_frequency1, pal_high_frequency1,
+        pal_user_id1)
+    ppa_record2, pal_records2 = makePpaAndPalRecordsConsistent(
+        ppa_record2, [pal_record_1], pal_low_frequency2, pal_high_frequency2,
+        pal_user_id2) 
+    
+    ppa_cluster_list_1 = [0]
+    ppa_cluster_list_2 = []
+    # Move device_b near to the first PPA zone
+    device_b['installationParam']['latitude'] = 38.82767
+    device_b['installationParam']['longitude'] = -97.20497
+
+    # Creating conditionals for Cat B devices
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    conditionalParameters = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['installationParam']
+
+    # N2 spectrum inquiry requests
+    # 1. Spectrum Inquiry: 3550-3700 .
+    spectrum_inquiry_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    spectrum_inquiry_1['inquiredSpectrum'] = [{
+        'lowFrequency': 3550000000,
+        'highFrequency': 3570000000
+    }]
```

Comment:
should this range be wider so we could get a PAL channel in the response if we move the CBSD in side the PPA?

Reply:
updated,   increased the channel size to '3580000000'